react/Labs/ExperimentalModal/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+
+import Dialog from '@material-ui/core/Dialog'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import DialogActions from '@material-ui/core/DialogActions'
+
+import withMobileDialog from '@material-ui/core/withMobileDialog'
+
+import AppTitle from '../../AppTitle'
+
+export const ExperimentalModal = props => {
+  const { dismissAction, fullScreen, children, open } = props
+  return (
+    <Dialog
+      open={open}
+      onClose={dismissAction}
+      fullScreen={fullScreen}
+      aria-labelledby="dialog-title"
```

Comment:
This shouldn't stay like that.

Reply:
Why that? `aria-labelledby`makes a reference to the node id = dialog-title. And this node is the modal title

Reply:
Oh right, my bad!